contracts/staking/stakeManager/StakeManager.sol

```diff
@@ -648,30 +684,22 @@ contract StakeManager is
         // find the rest of validators without signature
         unsignedCtx = _fillUnsignedValidators(unsignedCtx, address(0));
 
-        return
-            _increaseRewardAndAssertConsensus(
-                blockInterval,
-                proposer,
-                signedStakePower,
-                stateRoot,
-                unsignedCtx.unsignedValidators,
-                unsignedCtx.unsignedValidatorIndex,
-                unstakeCtx.deactivatedValidators,
-                unstakeCtx.validatorIndex
-            );
+        return _increaseRewardAndAssertConsensus(
+            blockInterval,
+            proposer,
+            signedStakePower,
+            stateRoot,
+            unsignedCtx.unsignedValidators,
+            unsignedCtx.unsignedValidatorIndex,
+            unstakeCtx.deactivatedValidators,
+            unstakeCtx.validatorIndex
+        );
     }
 
     function updateCommissionRate(uint256 validatorId, uint256 newCommissionRate) external onlyStaker(validatorId) {
         _updateRewards(validatorId);
 
-        delegatedFwd(
-            extensionCode,
-            abi.encodeWithSelector(
-                StakeManagerExtension(extensionCode).updateCommissionRate.selector,
-                validatorId,
-                newCommissionRate
-            )
-        );
+        delegatedFwd(extensionCode, abi.encodeWithSelector(StakeManagerExtension(extensionCode).updateCommissionRate.selector, validatorId, newCommissionRate));
     }
 
     function withdrawDelegatorsReward(uint256 validatorId) public onlyDelegation(validatorId) returns (uint256) {
```

Comment:
These functions haven't been changed - the formatter produced different results, so `@note`s were added above functions as a temporary fix to indicate which ones have been changed. 

For example:

```
    // @note topUpForFee
    function topUpForFee(address user, uint256 heimdallFee) public onlyWhenUnlocked {
        _transferAndTopUp(user, msg.sender, heimdallFee, 0, false);
    }

    // @note claimFee
    function claimFee(uint256 accumFeeAmount, uint256 index, bytes memory proof) public {
        _claimFee(accumFeeAmount, index, proof, false);
    }

    // @note
    function _claimFee(uint256 accumFeeAmount, uint256 index, bytes memory proof, bool legacy) internal {
        //Ignoring other params because rewards' distribution is on chain
        require(keccak256(abi.encode(msg.sender, accumFeeAmount)).checkMembership(index, accountStateRoot, proof), "Wrong acc proof");
        uint256 withdrawAmount = accumFeeAmount.sub(userFeeExit[msg.sender]);
        _claimFee(msg.sender, withdrawAmount);
        userFeeExit[msg.sender] = accumFeeAmount;
        _transferToken(msg.sender, withdrawAmount, legacy);
    }
```

That last `@note` is empty because it's related to the previous titled `@note` (`claimFee`).

It's worth noting that the only changes are adding `legacy` or passing `true`/`false` to functions with `legacy`.

This how VS Code displays it:

![image](https://github.com/0xPolygon/pos-contracts/assets/94782988/4c19c241-c861-415a-9938-a86773631334)

Reply:
All notes have now been removed to keep the code clean.